integration_tests/e2e/workAndSkillsPage.cy.ts

```diff
@@ -161,6 +161,7 @@ context('Work and skills page', () => {
           workAndSkillsPage.ES_header().should('exist')
           workAndSkillsPage.ES_header().contains('Employability skills')
           workAndSkillsPage.ES_heading().contains('Most recent levels')
+          workAndSkillsPage.ES_info().should('exist')
```

Comment:
I thought it would be best just to check that the element exists on the page, rather than the specific content of the element.

Reply:
💯 agreed 👍